### ISSUE TITLE
Official weights qwen2.5 vl oft metric mismatch

### ISSUE THREAD

**Issue by zhiwenhou1227** [starVLA/starVLA#236]
从hf[https://huggingface.co/StarVLA/Qwen2.5-VL-OFT-LIBERO-4in1/tree/main/checkpoints] 这里下载了权重进行评测，执行如下评测脚本

bash examples/LIBERO/eval_files/auto_eval_scripts/auto_eval_libero.sh

发现指标与官方无法对齐，主要是在一些任务差距很大，尤其是spatial和long任务

<img width="1140" height="572" alt="Image" src="https://github.com/user-attachments/assets/6b20b3c9-cbfe-49e7-8631-2cbcc5ca50f4" />
官方权重本地直接评测：
**object: 99.0
goal: 97.6
spatial: 91.6
long: 90.8
avg: 94.75**long和spatial任务要差很多

其中在spatial任务中，下面这个子任务sr尤其低
{
  "task_id": 5,
  "task_description": "pick up the black bowl on the ramekin and place it on the plate",
  "successes": 18,
  "episodes": 50,
  "success_rate": 0.36
}

另外，按照下面的参数进行训练，bs=16, num gpu = 8, 80k train steps
```json
datasets:
  vla_data:
    CoT_prompt: Your task is {instruction}. To identify the key objects for your task.
      Locate their bounding boxes in [x1,y1,x2,y2] format.
    data_mix: libero_all
    data_root_dir: playground/Datasets/LEROBOT_LIBERO_DATA
    dataset_py: lerobot_datasets
    per_device_batch_size: 16
    sequential_step_sampling: false
    video_backend: torchvision_av
framework:
  action_model:
    action_dim: 7
    action_hidden_dim: 2048
    action_model_type: DiT-B
    future_action_window_size: 7
    past_action_window_size: 0
  name: QwenOFT
  qwenvl:
    attn_implementation: flash_attention_2
    base_vlm: playground/Pretrained_models/Qwen2.5-VL-3B-Instruct
output_dir: xxxx
run_id: qwen2.5_vl_3b_oft_20260402
run_root_dir: xxx
seed: 42
trainer:
  eval_interval: 100
  freeze_modules: true
  gradient_accumulation_steps: 1
  gradient_clipping: 1.0
  is_resume: false
  learning_rate:
    action_model: 0.0001
    base: 3.0e-05
    qwen_vl_interface: 1.0e-05
  logging_frequency: 10
  lr_scheduler_type: cosine_with_min_lr
  max_train_steps: 80000
  num_warmup_steps: 5000
  optimizer:
    betas:
    - 0.9
    - 0.95
    eps: 1.0e-08
    weight_decay: 1.0e-08
  save_interval: 10000
  scheduler_specific_kwargs:
    min_lr: 1.0e-06
wandb_entity: jinhuiye
wandb_project: starVLA_Libero
```

评测30000 step的指标如下：
**object: 98.6
goal: 97.6
spatial: 90.4
long: 86.0
avg: 93.15**

不管是评测官方权重还是自己对齐参数训练后的权重目前都无法对齐指标

**Comment by JinhuiYE** [comment]
社区经验：从头 install libero 环境
